governance/sentinel-cidr-whitelist/README.md

```diff
@@ -0,0 +1,106 @@
+#  Vault Sentinel policy
+This guide shows how to enable a Sentinel policy in Vault that will only allow access to a secret enpoint if the request comes from a certain ip address.
+
```

Comment:
Spelling:  *endpoint

Also clarify, this is for a CIDR, which would allow for a network range or specific IP address.